src/pages/settings/PasswordPage.js

```diff
@@ -1,25 +1,158 @@
+import {
+    View,
+    TextInput,
+} from 'react-native';
+import Onyx, {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import {isEmpty} from 'underscore';
+
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import ROUTES from '../../ROUTES';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import Text from '../../components/Text';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import ButtonWithLoader from '../../components/ButtonWithLoader';
+import {changePassword} from '../../libs/actions/User';
+
+const propTypes = {
+    /* Onyx Props */
+    // Holds information about the users account that is logging in
+    account: PropTypes.shape({
+        // Whether 2FA is required for the users account that is logging in
+        requiresTwoFactorAuth: PropTypes.bool,
+
+        // An error message to display to the user
+        error: PropTypes.string,
+
+        // Whether or not a sign on form is loading (being submitted)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    account: {},
+};
+
+class PasswordPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentPassword: '',
+            newPassword: '',
+            confirmNewPassword: '',
+            twoFactorCode: '',
+        };
+
+        this.handleChangePassword = this.handleChangePassword.bind(this);
+    }
+
+    componentWillUnmount() {
+        Onyx.merge(ONYXKEYS.ACCOUNT, {error: ''});
+    }
+
+    handleChangePassword() {
+        changePassword(this.state.currentPassword, this.state.newPassword, this.state.twoFactorCode)
+            .then((response) => {
+                if (response.jsonCode === 200) {
+                    Navigation.navigate(ROUTES.SETTINGS);
+                }
+            });
+    }
 
-const PasswordPage = () => (
-    <ScreenWrapper>
-        {() => (
-            <>
-                <HeaderWithCloseButton
-                    title="Change Password"
-                    shouldShowBackButton
-                    onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
-                    onCloseButtonPress={() => Navigation.dismissModal()}
-                />
-            </>
-
-        )}
-    </ScreenWrapper>
-);
+    render() {
+        return (
+            <ScreenWrapper>
+                {() => (
+                    <>
+                        <HeaderWithCloseButton
+                            title="Change Password"
+                            shouldShowBackButton
+                            onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                            onCloseButtonPress={Navigation.dismissModal}
+                        />
+                        <View style={[styles.p5, styles.flex1]}>
+                            <Text fontSize={17} style={styles.mb6}>
```

Comment:
No need to increase the font size - let's just apply `styles.textP` and we should be set.